packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -162,18 +166,23 @@ export class FacetSearch {
   }
 
   private get showMoreSearchResults() {
-    if (this.props.controller.facetSearchState.values.length === 0) {
+    if (
+      this.props.controller.facetSearchState.values.length === 0 ||
+      !this.props.controller.facetSearchState.moreValuesAvailable
```

Comment:
I would guess that just `this.props.controller.facetSearchState.moreValuesAvailable` could be used? Do we really need the other condition ?